File: guides/micronaut-turbo-native/kotlin/src/main/kotlin/example/micronaut/advanced/SuccessController.kt
package example.micronaut.advanced

import io.micronaut.http.MediaType
import io.micronaut.http.annotation.Controller
import io.micronaut.http.annotation.Get
import example.micronaut.model.ViewModel
import io.micronaut.views.View

@Controller("/success")
class SuccessController {

    @View("success")
    @Get(produces = [MediaType.TEXT_HTML], consumes = [MediaType.TEXT_HTML])
    fun index() = ViewModel("It Worked!")
}

File: guides/micronaut-turbo-native/python/src/example/micronaut/advanced/success_controller.py
from micronaut.http import MediaType
from micronaut.http.annotation import Get, Produces
from micronaut.views import View

from ..model.view_model import ViewModel


@View("success")
@Get("/success")
@Produces(MediaType.TEXT_HTML)
def index() -> dict:
    return ViewModel("It Worked!").to_dict()
